packages/frontend/src/components/dialogs/ViewGroup.tsx

```diff
@@ -80,18 +80,16 @@ export const useGroup = (accountId: number, chat: T.FullChat) => {
     setGroupMembers(members => [...members, ...newGroupMembers])
   }
 
-  const reloadGroupContacts = async () => {
-    BackendRemote.rpc
-      .getContactsByIds(accountId, group.contactIds)
-      .then(contacts => {
-        // update contacts in case a contact changed
-        // while this dialog is open (e.g. contact got blocked)
-        setGroup(group => ({
-          ...group,
-          contacts: group.contactIds.map(id => contacts[id]),
-        }))
+  type GroupContacts = typeof group.contacts
+  const setGroupContacts = useCallback(
```

Comment:
```suggestion
  /**
   * Why setGroupContacts & setGroupMembers?
   * setGroupMembers triggers a modifiyGroup in the backend and is called after
   * changes triggered by the user while setGroupContacts is only called after
   * changes from "outside" (not on this client) after DCEvent "ContactsChanged"
   * to update the group in local state
   *
   * It takes a "setter" argument to make sure, the update always happens
   * on the latest group state
   */
  const setGroupContacts = useCallback(
```

Reply:
We should just get rid of `setGroupMembers`, it's buggy.
https://github.com/deltachat/deltachat-desktop/issues/5111#issuecomment-2883371449

But yeah, let's add the comment until then.